easybuild/tools/systemtools.py

```diff
@@ -406,7 +406,11 @@ def get_cpu_features():
                 # for Linux@POWER systems, no flags/features are listed, but we can check for Altivec
                 cpu_altivec_regex = re.compile("^cpu\s*:.*altivec supported", re.M)
                 if cpu_altivec_regex.search(proc_cpuinfo):
-                    cpu_feat = ['altivec']
+                    cpu_feat.append('altivec')
+                # VSX is supported since POWER7
+                cpu_power7_regex = re.compile("^cpu\s*:.*POWER(7|8|9)", re.M)
```

Comment:
For the "real" IBM POWER systems up until now, I believe yes. But I have no clue how things look like for other POWER-based designs (remember, there is the OpenPOWER initiative). Though other things in `systemtools` may break there as well...